app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
```

Comment:
But we cannot modify anything from Decidim's controllers, so cannot have the current_user/current_organization data until we make Decidim controllers inherit form the app's `ApplicationController` :(